plugin.video.srf_ch_replay/addon.py

```diff
@@ -87,12 +80,30 @@ def listTvShows():
             picture = show['Image']['ImageRepresentations']['ImageRepresentation'][0]['url']
         except:
             picture = ''
-        _addShow(title, show['id'], mode, desc, picture, page)
+
+        firstTitleLetter = title[:1]
+        if (firstTitleLetter.lower() == letter) or (not firstTitleLetter.isalpha() and not str(letter).isalpha()):
+            _add_show(title, show['id'], mode, desc, picture, page)
 
     xbmcplugin.addSortMethod(pluginhandle, 1)
     xbmcplugin.endOfDirectory(pluginhandle)
+    if myView:
```

Comment:
Not allowed sorry, plugins can't force skin viewmodes. Please drop this call.

Reply:
@enen92: Is there an allowed way for that? What I wanna basically do is to have the InfoWall ViewType when I'm in the add-on. Right now the default seems to be the list and for every new view or change a page I have to set the view type again. If there is a way could you please refer me to the documentation? Right now this was the only way I have found... Thank you for your help... if there is no way... I will remove it...

Reply:
I think there is no other way, the only option is to define a default view for your container content on the skin settings. That applies to all content including add-ons.

Reply:
@enen92: Is there an allowed way for that? What I wanna basically do is to have the InfoWall ViewType when I'm in the add-on. Right now the default seems to be the list and for every new view or change a page I have to set the view type again. If there is a way could you please refer me to the documentation? Right now this was the only way I have found... Thank you for your help... if there is no way... I will remove it...

Reply:
ok, then I remove it... thanks for the information